src/core/common/message.cpp

```diff
@@ -98,7 +98,7 @@ ThreadError Message::Init(void)
     return kThreadError_None;
 }
 
-Message *Message::New(uint8_t aType, uint16_t aReserved)
+Message *Message::New(uint8_t aType, uint16_t aReserved, bool aLinkSecurityEnabled)
```

Comment:
I thought link security was important enough to force the implementor to think about it when allocating a new message.  However, I would also be fine with having link security always `true` by default and removing the required argument from `Message::New`.  Would you prefer that?